### PR TITLE
Update WebGPU's webidl

### PR DESCRIPTION
**Pull request by JolifantoBambla**
I updated WebGPU's webidl from the current state of the spec.

However, I made one manual change to the IDL:
The constructor of [GPUPipelineError](https://www.w3.org/TR/webgpu/#gpupipelineerror) has an optional argument followed by a required argument, which webidl can't handle (or is illegal?). As a fix, I made the optional argument (`message`) a required one.
I think `GPUPipelineError` is not meant to be created by a user, so I'm not sure how big of a deal this is.

### CODE REVIEW

**Review by daxpedda — CHANGES_REQUESTED** [review]
LGTM!

Please remove the bump, this is something we will do separately.

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,13 @@
 * Extend `AudioContext` with unstable features supporting audio sink configuration.
   [#3433](https://github.com/rustwasm/wasm-bindgen/pull/3433)
 
+### Changed
+
+* Updated the WebGPU WebIDL.
+  The optional `message` argument of [GPUPipelineError](https://www.w3.org/TR/webgpu/#gpupipelineerror)'s constructor has been manually specified as a required argument,
```
Comment (daxpedda) [inline CHANGELOG.md:14]:
```suggestion
  The optional `message` argument of [`GPUPipelineError`](https://www.w3.org/TR/webgpu/#gpupipelineerror)'s constructor has been manually specified as a required argument,
```

**Comment by JolifantoBambla** [comment]
> LGTM!
> 
> Please remove the bump, this is something we will do separately.
> 
> EDIT: Please add a changelog entry as well.

Done :)

**Review by daxpedda — CHANGES_REQUESTED** [review]
Thank you, tiny nit left.

**Review by daxpedda — APPROVED** [review]
LGTM!

Thank you for your contribution!

**Pull request merged**